lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -295,6 +287,14 @@ defmodule Trento.Domain.SapSystem do
     }
   end
 
+  def execute(
```

Comment:
So this does nothing after all :laughing: 
We don't have any command afterward.
Anyway, leave it, maybe we need it in the future